src/book/03-guides/02-tools/02-webots.mdx

```diff
@@ -304,24 +277,33 @@ If you're running more than one robot, you will need to set up a Docker containe
 
 Follow these steps to set up a NUbots developed environment rather than the RoboCup environment.
 
-1. Move to the `NUWebots` folder:
+1. Clone the NUWebots repository:
+
+   ```bash
+   cd ~
+   git clone https://github.com/NUbots/NUWebots.git
+   ```
+
+2. Move to the `NUWebots` folder:
 
    ```bash
-   cd $WEBOTS_HOME/..
+   cd NUWebots
    ```
 
-2. Install the python dependencies:
+3. Install the python dependencies:
 
    ```bash
    pip3 install -r requirements.txt
    ```
 
-3. Configure the codebase by running:
+4. Configure the codebase by running:
 
    ```bash
    ./b configure
    ```
 
+   [Note: you must have `WEBOTS_HOME` set or this will fail](guides/tools/webots-setup#install-webots).
```

Comment:
Alert here too pls

Reply:
```suggestion
      <Alert type="warning">
      Note: [You must have `WEBOTS_HOME` set or this will fail](webots-setup#install-webots).
      </Alert>
```